### ISSUE TITLE
Add github.com as badge provider

### ISSUE THREAD

**Issue by andreujuanc** [microsoft/vscode-vsce#389]
Now with Github Actions having badges, we've got a new provider to add to the whitelist.
I'm removing my badges for now.

**Comment by joaomoreno** [comment]
Since https://github.com/microsoft/vscode-vsce/commit/84f537a4fac4b26b37a3cbd0f3644f0f517a04ec there is no more whitelist. Please update to the latest vsce.

**Comment by andreujuanc** [comment]
@joaomoreno  Well, when i had GitHub badges it stated that it was an invalid badge.

I will try again next week and create another issue if it occurs again.

Cheers

**Comment by andreujuanc** [comment]
@joaomoreno  
https://code.visualstudio.com/api/working-with-extensions/publishing-extension#publishing-extensions
https://code.visualstudio.com/api/references/extension-manifest#approved-badges

All these doc articles does not reflect the fact that the badges are not whitelisted anymore.

And this part of the doc , actually states the problem I have now:
`Images in README.md and CHANGELOG.md may not be SVGs unless they are from trusted badge providers.`

**Comment by joaomoreno** [comment]
@andreujuanc Thanks for the ping.

We've actually just had a discussion that this list should be brought back: https://github.com/microsoft/vscode-vsce/issues/395

I will fix it asap and release a new VSCE version.

**Comment by joaomoreno** [comment]
@andreujuanc Can you show me an example badge from GH?

**Comment by andreujuanc** [comment]
Sorry for the delay @joaomoreno it was a busy day.
Here is the badge I was trying to use:

`[![Actions Status](https://github.com/DigitalMolecules/vscode-servicebus-explorer/workflows/Build/badge.svg)](https://github.com/DigitalMolecules/vscode-servicebus-explorer/actions)`

Thanks for reopening this, it'd be really cool to show the badge :)

**Comment by joaomoreno** [comment]
We can't add that source as a trusted badge provider, since this would enable anyone to simply use any badge by simply uploading it to Github.

**Comment by joaomoreno** [comment]
Hm... I see that's actually the Actions badge. We might need to improve the pattern here, since we currently filter on domain only.

cc @viradhamMS @pkuma-msft

**Comment by JohnstonCode** [comment]
Reference to all the possible badge URLs
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository

**Comment by felangel** [comment]
@joaomoreno had to remove my badges as well 😞@joaomoreno do you know what security concerns warrant a whitelist? I haven't seen other package/plugin discovery platforms have this requirement.

**Comment by edgardmessias** [comment]
A temporary solution:
Instead of
`https://github.com/<:user>/<:repo>/workflows/<:workflow>/badge.svg`
Use the:
`https://img.shields.io/github/workflow/status/<:user>/<:repo>/<:workflow>.svg`

Example:
[![Build Status](https://github.com/JohnstonCode/svn-scm/workflows/build/badge.svg)](https://github.com/JohnstonCode/svn-scm/actions) vs [![Build Status](https://img.shields.io/github/workflow/status/JohnstonCode/svn-scm/build.svg)](https://github.com/JohnstonCode/svn-scm/actions)

Source: https://github.com/badges/shields/blob/c455993a4d307823a1f551955e9300cd3bc6e2c2/services/github/github-workflow-status.service.js

**Comment by Stanzilla** [comment]
@joaomoreno what's the long term plan here?

**Comment by joaomoreno** [comment]
It's to resolve the issue. 👍  Still waiting on the upstream Marketplace implementation.

**Comment by Stanzilla** [comment]
That's not a public issue we can follow, right?

**Comment by joaomoreno** [comment]
Unfortunately right, but I can tell you it's getting done!

**Comment by joaomoreno** [comment]
This is finally in! Both the server side changes and VSCE changes were deployed. Starting with vsce@1.77.0, you can now use github workflow badges.
